modules/core/src/main/java/org/apache/ignite/internal/managers/encryption/GridEncryptionManager.java

```diff
@@ -156,13 +201,13 @@ public GridEncryptionManager(GridKernalContext ctx) {
     @Override public void start() throws IgniteCheckedException {
         startSpi();
 
-        if (getSpi().masterKeyDigest() != null)
+        if (!ctx.clientNode() && getSpi().masterKeyDigest() != null)
```

Comment:
We don't need this attribute anymore.
Let's remove it completely.

Reply:
Done.